Makefile

```diff
@@ -18,3 +18,44 @@ wasm:
 
 parser:
 	go generate .
+
+build:
+	go build -ldflags=$(LDFLAGS) ./cmd/arrai
+
+
+######## Build information report
```

Comment:
The logic for collecting all this info seems a little shaky. Is it possible to just copy all the version stuff from https://github.com/anz-bank/sysl/blob/master/Makefile?